src/v/rpc/simple_protocol.cc

```diff
@@ -87,6 +87,12 @@ send_reply(ss::lw_shared_ptr<server_context_impl> ctx, netbuf buf) {
       });
 }
 
+ss::future<> send_reply_skip_payload(
+  ss::lw_shared_ptr<server_context_impl> ctx, netbuf buf) {
+    co_await ctx->res.conn->input().skip(ctx->get_header().payload_size);
+    co_await send_reply(std::move(ctx), std::move(buf));
+}
```

Comment:
nit: any particular reason we're leaving the `signal_body_parse()` call out?

Reply:
not for any particular reason except preserving the existing pattern where the caller was handling it in multiple other places.

Reply:
SGTM